src/app/app.component.ts

```diff
@@ -1,4 +1,5 @@
 import {Component} from '@angular/core';
+import {Router} from '@angular/router';
```

Comment:
mi riuscite a spiegare al volo cosa fa router esattamente? e perché l'avete messo qua dentro invece che nel modulo a parte?

Reply:
Collegato alla domanda prima non so perché sia qua, però ha lo scopo di gestire la navigazione tra le pagine cambiando l'URL in diversi Path senza invece restare sulla stessa pagina e cambiare solo il contenuto della finestra

Reply:
spiegato nella domanda di prima, in cui chiedevi perché è sparito